Source/m_menu.c

```diff
@@ -4027,12 +4027,10 @@ void M_InitExtendedHelp(void)
 	{
 	  if (extended_help_count)
 	  {
-	    if (gamemode == commercial)
-	      {
-		ExtHelpDef.prevMenu  = &ReadDef1; // previous menu
-		ReadMenu1[0].routine = M_ExtHelp;
-	      }
-	    else
+	    ExtHelpDef.prevMenu  = &HelpDef; // previous menu
```

Comment:
Why did you change the `if (gamemode == commercial) [...] else` logic here?

Reply:
Because both Doom 2 and Doom 1 display a help screen if the user presses F1. For some reason, extended help was only available from the "Read This!" menu in Doom 1.

Reply:
Please add a comment here.